client/scss/components/_panel.scss

```diff
@@ -45,6 +47,7 @@ $header-button-size: theme('spacing.8');
   align-content: center;
   color: theme('colors.primary.DEFAULT');
   border-radius: theme('borderRadius.full');
+  margin: 0;
```

Comment:
This is what has slightly changed the UI for InlinePanel – so we display the buttons without any space in-between. Even though the icons are only 16x16px, the buttons are 32x32px, so this big enough to not require the extra space.